Xamarin.Forms.Platform.UAP/ImageRenderer.cs

```diff
@@ -189,5 +195,39 @@ void UpdateAspect()
 				Element.SetIsLoading(false);
 			}
 		}
+
+		void UpdateImageSearchDirectory()
+		{
+			if (_disposed || Element == null || Control == null)
+			{
+				return;
+			}
+
+			var imageSearchDirectory = Application.Current.OnThisPlatform().GetImageSearchDirectory();
+
+			if(!string.IsNullOrEmpty(imageSearchDirectory))
+			{
+				var source = Element.Source;
+
+				if (source is FileImageSource)
+				{
+					var fileSource = source as FileImageSource;
+
+					if (fileSource != null)
+					{
+						var filePath = fileSource.File;
+
+						var directory = Path.GetDirectoryName(filePath);
+
+						if (string.IsNullOrEmpty(directory) || !Path.GetFullPath(directory).Equals(Path.GetFullPath(imageSearchDirectory)))
+						{
+							filePath = string.Format("{0}{1}{2}", imageSearchDirectory, Path.DirectorySeparatorChar, filePath);
```

Comment:
Why not `Path.Combine`?

Reply:
There is no reason and in fact, it is better. Changed. Thanks!.